File: PHP/UberX.php
<?php
require_once('Car.php');
class UberX extends Car{
    public $brand;
    public $model;

    public function __construct($license,$driver,$brand,$model){
        parent::__construct($license,$driver);
        $this->licence = $license;
        $this->driver = $driver;
    }

}

?>

File: Java/UberX.java
package Java;

class UberX extends Car {
    String brand;
    String model;

    public UberX(String licence,Account driver,String brand, String model){
        super(licence, driver);
        this.brand = brand;
        this.model = model;
        
        
    }
    
    @Override
    void printDataCar() {
        super.printDataCar();
        System.out.println("Model: " + model + " Brand: " + brand);
    }

}